### PR TITLE
Announce selected interns

### PR DESCRIPTION
**Pull request by wetneb**
Small post to announce selected interns and projects. Feel free to tweak.

### CODE REVIEW

File: _posts/2020-05-04-outreachy-gsoc-projects.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+We are happy to announce that three interns will join the development team this summer:
+
+* Ekta Mishra ([@darecoder1999](https://github.com/darecoder1999)), our Outreachy intern, will work on improving the quality assurance features of the Wikidata integration, by adding support for new Wikidata constraints and improving the existing
+  functionality;
+
+* Lisa Chandra ([@lisa761](https://github.com/lisa761)), Google Summer of Code intern, will improve our grid view by adding support for infinite scrolling (displaying the entire grid without the need to browse it by small pages);
+
+* Lu Liu ([afkbrb](https://github.com/afkbrb)), Google Summer of Code intern, will add support for OAuth and other Wikibase instances in the Wikidata extension.
```
Comment (tfmorris) [inline _posts/2020-05-04-outreachy-gsoc-projects.md:15]:
```suggestion
* Lu Liu ([@afkbrb](https://github.com/afkbrb)), Google Summer of Code intern, will add support for OAuth and other Wikibase instances in the Wikidata extension.
```

File: _posts/2020-05-04-outreachy-gsoc-projects.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+We are very excited to welcome them in the team and are looking forward to working with them.
```
Comment (tfmorris) [inline _posts/2020-05-04-outreachy-gsoc-projects.md:17]:
```suggestion
We are very excited to welcome them to the team and are looking forward to working with them.
```

**Comment by wetneb** [comment]
No strong feeling about the order, I guess I tend to put the lesser-known program first to give it more visibility - GSoC is very well known already indeed.

**Comment by magdmartin** [comment]
@wetneb  thanks for the draft. Looks good to me. 
Should we add more information like:
* who are each student mentors 
* the start and end date for each program.

**Pull request merged**